table/tables/index_test.go

```diff
@@ -381,28 +373,34 @@ func (s *testIndexSuite) TestMultiColumnCommonHandle(c *C) {
 			Ft:         rowcodec.FieldTypeFromModelColumn(b),
 		})
 		colVals, err := tablecodec.DecodeIndexKV(key, val, 1, tablecodec.HandleDefault, colInfo)
-		c.Assert(err, IsNil)
-		c.Assert(colVals, HasLen, 3)
+		require.Nil(t, err)
+		require.Len(t, colVals, 3)
 		_, d, err := codec.DecodeOne(colVals[0])
-		c.Assert(err, IsNil)
-		c.Assert(d.GetString(), Equals, "abc")
+		require.Nil(t, err)
+		require.Equal(t, "abc", d.GetString())
 		_, d, err = codec.DecodeOne(colVals[1])
-		c.Assert(err, IsNil)
-		c.Assert(d.GetInt64(), Equals, int64(3))
+		require.Nil(t, err)
+		require.Equal(t, int64(3), d.GetInt64())
 		_, d, err = codec.DecodeOne(colVals[2])
-		c.Assert(err, IsNil)
-		c.Assert(d.GetInt64(), Equals, int64(2))
+		require.Nil(t, err)
+		require.Equal(t, int64(2), d.GetInt64())
 		handle, err := tablecodec.DecodeIndexHandle(key, val, 1)
-		c.Assert(err, IsNil)
-		c.Assert(handle.IsInt(), IsFalse)
-		c.Assert(handle.Encoded(), BytesEquals, commonHandle.Encoded())
+		require.Nil(t, err)
+		require.False(t, handle.IsInt())
+		require.Equal(t, commonHandle.Encoded(), handle.Encoded())
 	}
 }
 
-func buildTableInfo(c *C, sql string) *model.TableInfo {
+func buildTableInfo(t *testing.T, sql string) *model.TableInfo {
 	stmt, err := parser.New().ParseOneStmt(sql, "", "")
-	c.Assert(err, IsNil)
+	require.Nil(t, err)
 	tblInfo, err := ddl.BuildTableInfoFromAST(stmt.(*ast.CreateTableStmt))
-	c.Assert(err, IsNil)
+	require.Nil(t, err)
 	return tblInfo
 }
+
+func TestTearDownSuite(t *testing.T) {
```

Comment:
teardown should not be a test, you should batch setup logic and use when necessary